### PR TITLE
Jira structure: back to swagger schemas, but fix incorrect

### LINKED ISSUE #5256: Jira Connector: "issues" stream structure changed unexpectedly
## Enviroment
- **Airbyte version**: 0.28.2-alpha
- **OS Version / Instance**:  AWS EC2
- **Deployment**: Docker
- **Source Connector and version**: Jira 0.2.8 
- **Destination Connector and version**: Postgres 0.3.7 
- **Severity**: Medium
- **Step where error happened**: Update connector

## Current Behavior
The stream `issues` used to contain a lot more information in the `fields` object. 
Attributes like: `project`, `statuscategorychangedate`, `summary` and others, are missing.
I'm using Grafana to query through data and create dashboards and since those attributes are now missing, my dashboards are broken.

I've tried to downgrade the connector to version 0.2.7, but no success.

## Expected Behavior
The stream `issues` should return all issue information (just as the [Jira Rest API](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/#api-rest-api-3-issue-issueidorkey-get) provides).
With that behaviour we won't need to perform join queries to retrieve complementary issue data.

## Steps to Reproduce
1. Create a Jira connection
2. Perform a sync (with the `issues` stream selected)
3. Try to retrieve the project that issue belongs to:
    ```sql
    SELECT * FROM issues I
    WHERE I.fields -> 'project' ->> 'key' = '<your-project-key>';
    ```

## Are you willing to submit a PR?

No.

### PR DESCRIPTION
**Pull request by vovavovavovavova**
## What
Closes #5256 

## How
1) returned to previous schemas (discard auto-generated)
2) fixes: schemas, that had "items" root caused https://github.com/airbytehq/airbyte/issues/4950 now have properties root
3) repaired some files like airbyte-integrations/connectors/source-jira/source_jira/schemas/avatars.json
It's schema, had both system and custom roots, when in the code we expand and get inner value. Now schema suits to output
4) add additionalproperties, where response contains.more fields, than written in schema. Some original schemas had additional properties as false, that caused error because  of some fields.
5) add null values
6) remove parse_response_root from some classes. As example, `class IssueVotes`. Using this parse_response_root returns only the voters (users), but we need also the information about the votes https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-votes/#api-rest-api-3-issue-issueidorkey-votes-get . We skipped info hasVoted, votes amount, self. Those schemas and roots were repaired to return full information, not only voters. Several changes in streams have the same reason.
7) some other issues were fixed to pass the normalization through Airbyte UI

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> New Connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] `docs/SUMMARY.md`
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector added to connector index like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
   
</p>
</details>

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</p>
</details>

<details><summary> <strong> Connector Generator </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-jira/source_jira/schemas/avatars.json [hunk 1]
```diff
@@ -1,35 +1,47 @@
...
-      "type": "object"
+      "description": "The list of avatar icon URLs.",
+      "readOnly": true
     }
   },
-  "type": "object"
-}
+  "additionalProperties": false,
+  "description": "List of system avatars."
+}
```
Comment (vovavovavovavova) [inline airbyte-integrations/connectors/source-jira/source_jira/schemas/avatars.json:47]: todo add newline

File: airbyte-integrations/connectors/source-jira/source_jira/schemas/labels.json [hunk 2]
```diff
@@ -1,33 +1,24 @@
 {
-  "$schema": "http://json-schema.org/schema#",
+  "type": ["object", "null"],
```
Comment (gaart) [inline airbyte-integrations/connectors/source-jira/source_jira/schemas/labels.json:2]: Should there be a
`"$schema": "http://json-schema.org/draft-07/schema#",` 
too?

**Review by gaart — APPROVED** [review]

**Review by vitaliizazmic — COMMENTED** [review]
Could you provide some details about changes?

**Comment by vovavovavovavova** [comment]
> Could you provide some details about changes?

upgraded description

File: airbyte-integrations/connectors/source-jira/source_jira/schemas/labels.json (hunk 2 above)
Comment (vovavovavovavova) [reply airbyte-integrations/connectors/source-jira/source_jira/schemas/labels.json:2]: autogenerated schema always have this entry `$schema`. The swagger big hairy json is https://developer.atlassian.com/cloud/jira/platform/swagger.v3.json . Accordingly to it, the labels are https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-labels/#api-group-labels PageBeanString. In this json document it is described here `root['components']['schemas']['PageBeanString']` and it just does not contain $schema entry. Schemas were restored to the state before https://github.com/airbytehq/airbyte/pull/4947 and it seems that schemas, as well swagger one's just does not contain this entry. We can modify existing jsons and set it, BUT the previous version and swagger do not have it, and we use swagger.

**Review by vitaliizazmic — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

**Pull request merged**